packages/tools/create-kadena-app/scripts/generate-test.mjs

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env zx
+
+import 'zx/globals';
+
+// This script is used to generate projects for every supported template and test them.
+const generateAndTest = (template) =>
+  within(async () => {
+    const directory = `test-${template}-project`;
+    // echo "Generating the project"
+    await $`node lib/index.js generate-project --name ${directory} --template ${template}`;
```

Comment:
does the (forward) slash work in windows? I think we might want `path.join()` here instead

Reply:
agreed that we must be aware to be explicitly os agnostic but in my view this point is perpendicular to the tool being used. With regards to this, `zx` is the same as execa or spawning a process with node as far as I can tell

Reply:
I have updated the code.
@webpro that was an important tip, I will make sure everything is working as expected on Mac, Windows and WSL